add event information for Southampton seminar (#24)
UNIVERSE-HPC/UNIVERSE-HPC.github.io: _pages/events/2024-07-05_seminar-southampton.md

---
layout: single
classes: wide
title: UNIVERSE-HPC Project Seminar Series
permalink: /events/2024-07-05_seminar-southampton/
author_profile: false

---

## Navigating the Challenges and Successes of Delivering Training at a University

**Join us for the next UNIVERSE-HPC seminar on Friday 5 July at
14:00 BST**, which will take place **online**.

### [Register now](https://forms.office.com/e/PfEDWPdFk5).

## Overview:

- 🕒 **Friday 5th July** 2024, **[14:00 - 15:00 BST](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Navigating+the+Challenges+and+Successes+of+Delivering+Training+at+a+University&iso=20240705T14&p1=136&ah=1)**
- 🌍 Online via Zoom

> - 🗣️ **Philippa Broadbent**, Research Software Engineer at Southampton Research Software Group.

Reviewer: Do you want to note include "University of Southampon"?
```suggestion
- 🗣️ **Philippa Broadbent**, Research Software Engineer at [Southampton Research Software Group](https://rsgsoton.net/), University of Southampton
```

Running training at a university presents a diverse set of challenges, from securing funding, to teaching academics of all disciplines, to scaling up provision in line with demand. This talk will discuss some of the obstacles we’ve encountered while running training in computational research skills at the University of Southampton, drawing on my experience as the current training lead for the Southampton Research Software Group, and also the experiences of other long-standing members of the group.  After talking through the history of the group’s training provision, from the first instances of Software Carpentry training run at Southampton back in 2015 to our current training offerings, I’ll discuss the various difficulties we’ve encountered and the solutions we’ve implemented to address them.

Please read [our event privacy notice](https://www.imperial.ac.uk/media/imperial-college/administration-and-support-services/secretariat/public/ICL---Events-privacy-notice---10-October-2018.pdf).

> **Code of conduct**: This event takes place under our [Code of Conduct]({{ site.baseurl }}/events/code-of-conduct).

Reviewer: ```suggestion
**Code of conduct**: This event takes place under the UNIVERSE-HPC [Code of Conduct]({{ site.baseurl }}/events/code-of-conduct).
```

#### Join our email list

To stay in touch with our project activities and plans you can subscribe to our
UNIVERSE-HPC project mailing list:

<a
href="mailto:sympa@mlist.is.ed.ac.uk?body=SUBSCRIBE%20universe-hpc%20FIRSTNAME%20SURNAME%20%0A%0AQUIT%0A%0A">Subscribe
via email</a> <small>_(Clicking this link will create a new email to send a
subscription request to join the mailing list. Leave the email subject blank
and replace FIRSTNAME and SURNAME with your firstname and surname and then send
the email to join the list.)_</small>

<small>You can find [further
information](https://www.ed.ac.uk/information-services/computing/comms-and-collab/email/lists/sympa/subscribe)
on subscribing to a "Sympa" mailing list on the University of Edinburgh
website.</small>

> If you have any problems joining the mailing list please contact us at
> [our email](mailto:s.sukhiani@epcc.ed.ac.uk) to be added manually.

Reviewer: ```suggestion
If you have any problems joining the mailing list please
[contact us](mailto:s.sukhiani@epcc.ed.ac.uk) to be added manually.
```
